### ISSUE TITLE
meepo.pem is cat't use

### ISSUE THREAD

**Issue by loveoranges** [PeerXu/meepo#17]
```
C:\bin\meepo>meepo serve -f meepo.pem
Error: unknown shorthand flag: 'f' in -f
```

**Comment by PeerXu** [comment]
Thanks!
It's a typo on writing readme.
Try `-i` instead.

**Comment by PeerXu** [comment]
Fix readme typo #19

**Comment by PeerXu** [comment]
Problem solved, if any with this issue, feel free to reopen.
